client/client_test.go

```diff
@@ -193,32 +195,52 @@ func TestAddLeaf(t *testing.T) {
 	cli := trillian.NewTrillianLogClient(env.ClientConn)
 	for _, test := range []struct {
 		desc    string
+		leaf    []byte
 		client  trillian.TrillianLogClient
```

Comment:
with the current test data you could just drop the `client` member and use `cli`